src/lib/queries/getAccountsTotals.ts

```diff
@@ -4,23 +4,24 @@ import Money from '@/book/Money';
 import { Split } from '@/book/entities';
 import type { Account } from '@/book/entities';
 import mapAccounts from '@/helpers/mapAccounts';
-import aggregateChildrenTotals from '@/helpers/aggregateChildrenTotals';
 import type { AccountsTotals } from '@/types/book';
-import type { PriceDBMap } from '@/book/prices';
 
+/**
+ * Returns the sum of splits amounts for each account
+ * up until the selected date
+ */
 export default async function getAccountsTotals(
   accounts: Account[],
-  prices: PriceDBMap,
   selectedDate: DateTime,
 ): Promise<AccountsTotals> {
   const rows: { total: number, accountId: string, mnemonic: string }[] = await Split
     .query(`
       SELECT
-        ABS(SUM(cast(splits.quantity_num as REAL) / splits.quantity_denom)) as total,
```

Comment:
This was a bug. Some accounts does make sense to have `ABS` like Income/Expense but some accounts may have both negative or positive amounts when calculating partial totals like liabilities/assets (they have debit/credit)